pkg/rhttp/datatx/manager/tus/tus.go

```diff
@@ -99,7 +98,8 @@ func (m *manager) Handler(fs storage.FS) (http.Handler, error) {
 	config := tusd.Config{
 		StoreComposer:         composer,
 		NotifyCompleteUploads: true,
-		Logger:                log.New(appctx.GetLogger(context.Background()), "", 0),
+		// TODO: bring back!
```

Comment:
Is that on purpose?

Reply:
Good Question. The problem is that the logger interface changed in v2. It now uses https://pkg.go.dev/golang.org/x/exp/slog Afaikt there a two option how to solve it

A) We adjust our logger to follow new interface. I think we'd need to implement https://pkg.go.dev/golang.org/x/exp/slog#Handler

B) We use their default logger.

Is it worth going for A?

Reply:
Brought back the logger - but it is still noop.